modules/system/console/ThemeSync.php

```diff
@@ -181,7 +181,9 @@ public function handle()
         }
 
         // Confirm with the user
-        if (!$this->confirmToProceed(sprintf('This will OVERWRITE the %s provided paths in "themes/%s" on the %s with content from the %s', count($validPaths), $themeName, $target, $source), function () { return true; })) {
+        if (!$this->confirmToProceed(sprintf('This will OVERWRITE the %s provided paths in "themes/%s" on the %s with content from the %s', count($validPaths), $themeName, $target, $source), function () {
+            return true;
+        })) {
```

Comment:
@bennothommo this is more clear being a one line closure. According to [the docs](https://laravel.com/api/5.5/Illuminate/Console/ConfirmableTrait.html#method_confirmToProceed) it supports a bool argument so this closure can just be replaced by `true`.

Reply:
@LukeTowers I actually would prefer it being out of the `if` statement altogether, but I've changed it to `true` as per your recommendation, just to get the ball rolling.